introduction.md

```diff
@@ -3,3 +3,8 @@ Hi my name is Ariel Borochov
 I want to learn git
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?